templates/bot/js/command-and-response/README.md

```diff
@@ -74,24 +74,25 @@ The default pattern to trigger a command is through a defined keyword. But often
 
 When using regular expressions, any capture group can be found in `message.matches`. Below is an example that uses regular expression to capture strings after `reboot`, for example if user inputs `reboot myMachine`, `message.matches[1]` will capture `myMachine`:
 
-```typescript
-export class HelloWorldCommandHandler implements TeamsFxBotCommandHandler {
-  triggerPatterns: TriggerPatterns = /^reboot (.*?)$/i; //"helloWorld";
-  async handleCommandReceived(
-    context: TurnContext,
-    message: CommandMessage
-  ): Promise<string | Partial<Activity> | void> {
+```javascript
+class HelloWorldCommandHandler {
+  triggerPatterns = /^reboot (.*?)$/i; //"helloWorld";
+  async handleCommandReceived(context, message) {
     console.log(`Bot received message: ${message.text}`);
     const machineName = message.matches[1];
     console.log(machineName);
     // Render your adaptive card for reply message
-    const cardData: CardData = {
+    const cardData = {
       title: "Your Hello World Bot is Running",
       body: "Congratulations! Your hello world bot is running. Click the button below to trigger an action.",
     };
-    return MessageBuilder.attachAdaptiveCard<CardData>(helloWorldCard, cardData);
+    return MessageBuilder.attachAdaptiveCard(helloWorldCard, cardData);
```

Comment:
Let's remove `MessageBuilder` and use `MessageFactory` instead since we will deprecate this API in the future.
See https://github.com/OfficeDev/TeamsFx/pull/6287 for more details.

```javascript
const cardJson = AdaptiveCards.declare(helloWorldCard).render(cardData);
return MessageFactory.attachment(CardFactory.adaptiveCard(cardJson));
```